L1Trigger/L1THGCal/src/HGCalTriggerTowerGeometryHelper.cc

```diff
@@ -19,7 +19,10 @@ HGCalTriggerTowerGeometryHelper::HGCalTriggerTowerGeometryHelper(const edm::Para
       nBinsEta_(conf.getParameter<int>("nBinsEta")),
       nBinsPhi_(conf.getParameter<int>("nBinsPhi")),
       binsEta_(conf.getParameter<std::vector<double> >("binsEta")),
-      binsPhi_(conf.getParameter<std::vector<double> >("binsPhi")) {
+      binsPhi_(conf.getParameter<std::vector<double> >("binsPhi")),
+      splitModuleSum_(conf.getParameter<bool>("splitModuleSum")),
+      splitDivisorSilic_(conf.getParameter<int>("splitDivisorSilic")),
+      splitDivisorScint_(conf.getParameter<int>("splitDivisorScint")){
```

Comment:
The compatibility of the splitDivisors with the towerShare values in the mapping file needs to checked. Otherwise one could possibly get fractions larger than 1.

Ideally the splitDivisors should be written in and extracted from the mapping file. This way everything is consistently in one place, and it simplifies the configuration.